### DISCUSSION TITLE
Github copilot subscription not working

### DISCUSSION THREAD

**Discussion by Antony051** [community/community#43678] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have been using copilot since the technical preview and after it ended I have been using copilot as a verified student. But I am not able to use copilot from today. i am seeing that my github copilot technical preview has ended and i should subcribe. 
![image](https://user-images.githubusercontent.com/65231030/211242158-4da31727-fc92-4d55-be56-0676add561fd.png)

How do i start using copilot again?

**Comment by otkrickey** [comment]
Solved: #43634
